scripts/dev_setup.py

```diff
@@ -25,13 +25,15 @@ def exec_command(command):
 print('Running dev setup...')
 print('Root directory \'{}\'\n'.format(root_dir))
 
+# install general requirements
+exec_command('pip install -r requirements.txt')
```

Comment:
Since we don't have the keyvault problem anymore, I have no issue with it as is. :)